README.md

```diff
@@ -237,6 +239,11 @@ NLP as API with higher level functionality such as NER, Topic tagging and so on
 - [rstWeb](https://corpling.uis.georgetown.edu/rstweb/info/) - open source local or online tool for discourse tree annotations
 - [GitDox](https://corpling.uis.georgetown.edu/gitdox/) - open source server annotation tool with GitHub version control and validation for XML data and collaborative spreadsheet grids
 
+### Libraries and Tooling
+
+* [Transformers](https://github.com/huggingface/transformers) - Natural Language Processing for TensorFlow 2.0 and PyTorch
+* [Tokenizers](https://github.com/huggingface/tokenizers) - Tokenizers optimized for Research and Production
+
```

Comment:
Err, the bulk of this repo is libraries and tooling. It calls them out as "Libraries"


Please add these to the Python list itself?

Reply:
Sure thing!